.env.development

```
@@ -0,0 +1 @@
+VITE_API_URL=http://localhost:8080/api
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)